index.html

```diff
@@ -36,10 +36,10 @@ <h2 class="input-player-name-title">Enter Names To Start Game</h2>
       </div>
       <table class='game-question-table'>
         <tr>
-          <td class="title-category1 category1">Category1</td>
-          <td class="title-category2 category2">Category2</td>
-          <td class="title-category3 category3">Category3</td>
-          <td class="title-category4 category4">Category4</td>
+          <td class="dan title-category1 category1">Category1</td>
```

Comment:
What is the class of "dan" doing? Worried that the name of you class will cause issues for you further down the road as your codebase grows.